config/dictionaries/resource.json

```diff
@@ -4,16 +4,27 @@
     "name": "Application",
     "fields": [
       "id",
-      "user",
+      "created_at",
+      "is_host",
+      "should_track_latest_release",
+      "is_public",
+      "organization",
+      "application_config_variable",
+      "application_environment_variable",
+      "application_tag",
+      "is_depended_on_by__application",
+      "is_archived",
+      "is_for__device_type",
       "owns__device",
-      "depends_on__application",
-      "actor",
+      "owns__release",
       "app_name",
       "slug",
-      "commit",
+	  "commit",
+	  "should_be_running__release",
       "application_type",
-      "device_type",
-      "should_track_latest_release",
+	  "should_track_latest_release",
+	  "user__is_member_of__application",
```

Comment:
There's some white-space issue here.

Reply:
Converted all to spaces, will check this bit after pushing to GitHub.